tests/misc/script/tscript.nims

```diff
@@ -106,3 +65,14 @@ when false:
 
 rmDir("tempXYZ")
 doAssert dirExists("tempXYZ") == false
+
+import std/strtabs
+block ensure_strtab_interpolation_works_in_scripting_env:
+  # the strtab module relied on `os.getEnv` explicitly, now it's not longer
```

Comment:
`std/strtabs` depending on the `os` module feels ridiculous. Given that only the compiler seems to use the "default to environment variable" feature, maybe the `flags` should be replaced with a callback.

Reply:
Put in a comment indicating that this is on the way out.